src/App.tsx

```diff
@@ -17,18 +62,29 @@
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                visible={visible}
```

Comment:
Ensure that the `visible` prop type matches the expected `Visible` type. If `Visible` is not a string, this might cause a type mismatch.